cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -110,6 +125,12 @@ def _cleanup():
      delay=5, num_sec=2700)
     assert (request_details_list.is_successful(vms[0]) and
         not request_details_list.is_errored(vms[0]))
+    # validate MAC address matches between source and target VMs
+    src_vm = form_data_vm_obj_single_network[1][0]
+    rhv_prov = v2v_providers[1]
+    collection = rhv_prov.appliance.provider_based_collection(rhv_prov)
+    migrated_vm = collection.instantiate(src_vm.name, rhv_prov)
+    assert src_vm.mac_address == migrated_vm.mac_address
```

Comment:
Seems like this code is duplicated in a couple places, thoughts on breaking it out into a single method?

Reply:
Fixed it, check it out @jawatts :)